### PR TITLE
feat(ui/graph): enhance task status visibility

### LINKED ISSUE #34455: Graph view task name & status visibility
### Description

I have had complaints from coworkers that it is harder to visibly see the status of airflow tasks in the new graph view. They miss the colored border from the old graph view that made it very clear what the status of a task was. They have also mentioned that names of the tasks feel a lot smaller and harder to read without zooming in.

### Use case/motivation

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by hainenber**
Closes #34455 

Enhance DAG's task visibility by adding border color to task's node and increase size for task's status note.

# UI change

Before
Screenshot 2023-09-20 at 21 50 28

After
Screenshot 2023-09-20 at 00 02 03

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by bbovenzi** [comment]
Thanks @hainenber Could you provide some before and after screenshots?

**Review by hussein-awala — APPROVED** [review]
Looks good!

File: airflow/www/static/js/dag/details/graph/Node.tsx [hunk 1]
```diff
@@ -93,6 +93,11 @@ export const BaseNode = ({
     [, operatorTextColor] = labelStyle.split(":");
   }
 
+  const nodeBGColor =
```
Comment (bbovenzi) [inline airflow/www/static/js/dag/details/graph/Node.tsx]:
```suggestion
  const nodeBorderColor =
```

This is a border not a background.

File: airflow/www/static/js/dag/details/graph/Node.tsx [hunk 2]
```diff
@@ -107,8 +112,8 @@ export const BaseNode = ({
     >
       <Box
         borderRadius={5}
-        borderWidth={1}
-        borderColor={isSelected ? "blue.400" : "gray.400"}
+        borderWidth={1.5}
+        borderColor={isSelected ? "blue.400" : nodeBGColor}
```
Comment (bbovenzi) [inline airflow/www/static/js/dag/details/graph/Node.tsx]:
```suggestion
        borderColor={isSelected ? "blue.400" : nodeBorderColor}
```

**Comment by hainenber** [comment]
I think your proposal works pretty fine. I've made a draft change and the selected task looks identifiable re: its status here. Do let me know what you think

Screenshot 2023-09-21 at 23 36 13

**Comment by hainenber** [comment]
@bbovenzi does this look good to you? 👀 Sorry if it appears I'm nagging you.

**Review by bbovenzi — APPROVED** [review]

**Pull request merged**